Using-the-API/API.md

```diff
@@ -640,6 +641,14 @@ ___
 | `ancestors`              | The ancestors of the status in the conversation, as a list of [Statuses](#status)   | no       |
 | `descendants`            | The descendants of the status in the conversation, as a list of [Statuses](#status) | no       |
 
+### Emoji
+
+| Attribute                | Description                        | Nullable |
+|--------------------------|------------------------------------|----------|
+| `shortcode`              | The shortcode of the emoji         | no       |
+| `static_url`             | URL to the emoji static image      | no       |
+| `url`                    | URL to the emoji image             | no       |
```

Comment:
You might want to be a bit more clear about the difference between static_url and url here. maybe something like "URL to the emoji image (may be animated)" for `url` and "URL to the static emoji image, with animation stripped (auto-generated)"?